ffig/templates/d.tmpl

```diff
@@ -0,0 +1,10 @@
+{% for class in classes -%}
+class {{class.name}}
+{
+{% for method in class.methods %}
+  void {{method.name}}()
+  {
+  }
+{% endfor %}
```

Comment:
you can use `{-%` and `%-}` to swallow preceeding or following whitespace. 
This should get the test to pass.

Reply:
I think the `{%-` and `-%}` change would be better addressed at the top level (maybe with `trim_blocks`).